controllers/storagecluster/volumesnapshotterclasses.go

```diff
@@ -70,24 +72,76 @@ func (r *StorageClusterReconciler) createSnapshotClasses(vsccs []SnapshotClassCo
 	return nil
 }
 
+func (r *StorageClusterReconciler) getClusterIDAndSecretName(instance *ocsv1.StorageCluster, snapshotType util.SnapshotterType) (string, string, error) {
+	clusterID := instance.Namespace
+	secretName := fmt.Sprintf("rook-csi-%s-provisioner", snapshotType)
+
+	if !instance.Spec.ExternalStorage.Enable {
+		return clusterID, secretName, nil
+	}
+
+	data, ok := externalOCSResources[instance.UID]
+	if !ok {
+		err := fmt.Errorf("unable to retrieve external resource from externalOCSResources")
+		log.Error(err,
+			"unable to generate name for snapshot class secret for external mode")
```

Comment:
We dont need 2 lines for this.

```suggestion
		log.Error(err, "unable to generate name for snapshot class secret for external mode")
```